ida-ws/src/main/java/upb/ida/logging/AspectLogger.java

```diff
@@ -0,0 +1,137 @@
+package upb.ida.logging;
+
+/**
+ * Logs Creation - Using Spring AOP to provide universal logging of all the
+ * exceptions and method calls.
+ * 
+ * @author Ayaz
+ *
+ */
+
+import javax.servlet.http.HttpServletRequest;
+import org.aspectj.lang.JoinPoint;
+import org.aspectj.lang.annotation.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import upb.ida.bean.ResponseBean;
+import upb.ida.constant.IDALiteral;
+import org.apache.log4j.BasicConfigurator;
+
+@Component
+@Aspect
+public class AspectLogger 
+{
+	@Autowired(required=true)
+	private HttpServletRequest request;
+	@Autowired
+	private ResponseBean response;
+	@Autowired
+	private LoggerProvider loggerProvider;
```

Comment:
don't autowire the loggerprovider, autowire the logger itself. 
it should look something like this:
```
@Autowired
@Qualifier("requestResponseLogger")
Logger reqRespLogger;
```